src/main/java/io/codelex/oop/Cars/Car.java

```diff
@@ -0,0 +1,94 @@
+package io.codelex.oop.Cars;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private String name;
+    private String model;
+    private int price;
+    private int yearOfManufacture;
+    private List<Manufacturer> manufacturerList = new ArrayList();
+    private EngineType engineType;
+
+    public Car(String name, String model, int price, int yearOfManufacture, EngineType engineType) {
+        this.name = name;
+        this.model = model;
+        this.price = price;
+        this.yearOfManufacture = yearOfManufacture;
+        this.engineType = engineType;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = price;
+    }
+
+    public int getYearOfManufacture() {
+        return yearOfManufacture;
+    }
+
+    public void setYearOfManufacture(int yearOfManufacture) {
+        this.yearOfManufacture = yearOfManufacture;
+    }
+
+    public List<Manufacturer> getManufacturerList() {
+        return manufacturerList;
+    }
+
+    public void addManufacturerList(Manufacturer manufacturer) {
+        manufacturerList.add(manufacturer);
+    }
+
+    public EngineType getEngineType() {
+        return engineType;
+    }
+
+    public void setEngineType(EngineType engineType) {
+        this.engineType = engineType;
+    }
+
+    @Override
+    public String toString() {
+        return "Car{" +
+                "name='" + name + '\'' +
+                ", model='" + model + '\'' +
+                ", price=" + price +
+                ", yearOfManufacture=" + yearOfManufacture +
+                ", engineType=" + engineType +
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Car car = (Car) o;
+        return Double.compare(car.price, price) == 0 && yearOfManufacture == car.yearOfManufacture && Objects.equals(name, car.name) && Objects.equals(model, car.model) && Objects.equals(manufacturerList, car.manufacturerList) && engineType == car.engineType;
```

Comment:
if you have to scroll to the left, the line is too long 😄
https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html

Reply:
Salabots!